setup.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Setup file for easy installation."""
+import glob
+
+from rpmlint import __version__
+from setuptools import setup
+
+setup(
+    name='rpmlint',
+    description='RPM file QA correctness validator',
+    long_description='Command-line tool for RPM files QA validation',
+    url='https://github.com/rpm-software-management/rpmlint',
+    download_url='https://github.com/rpm-software-management/rpmlint',
+
+    version=__version__,
+
+    author='Frédéric Lepied',
+    author_email='flepied@mandriva.com',
+
+    maintainer='Neal Gompa',
+    maintainer_email='ngompa13@gmail.com',
+
+    license='License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)',
+    classifiers=[
+        # complete classifier list:
+        # http://pypi.python.org/pypi?%3Aaction=list_classifiers
+        'Development Status :: 5 - Production/Stable',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)',
+        'Operating System :: Unix',
+        'Operating System :: POSIX',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
```

Comment:
Intentionally leaving out the Python :: 2.7 marker? ;)

Reply:
The 3 is to be used if it works with any 3. you are supposed to diferentiate only if you can reasonably say it works only with ie 3.5+